convert2rhel/unit_tests/checks_test.py

```diff
@@ -1206,10 +1213,10 @@ def test_is_loaded_kernel_latest_skip_on_not_latest_ol(pretend_os, monkeypatch,
             "The currently loaded kernel is at the latest version.",
         ),
         (
-            "1634146676\t3.10.0-1160.45.1.el7\tbaseos",
+            "1634146676\tkernel-3.10.0-1160.45.1.el7\tbaseos",
             "3.10.0-1160.45.1.el7.x86_64",
             0,
-            6,
+            7,
             "kernel",
             False,
             "The currently loaded kernel is at the latest version.",
```

Comment:
We already have a major version `7` on the parameter above, I think it's safe for you to remove this altogether.

Reply:
Noted